src/AWS.Distro.OpenTelemetry.AutoInstrumentation/AwsSpanProcessingUtil.cs

```diff
@@ -111,6 +114,14 @@ internal static string GetIngressOperation(Activity span)
         {
             return InternalOperation;
         }
+
+        // There is the case where we can run an ASP.NET application in a lambda environment.
+        // In this case, we need to decide which takes precedence for the aws.local.operation
+        // between functionName/FunctionHandler or the route template.
+        else if (IsLambdaEnvironment())
+        {
+            return AwsLambdaFunctionName + "/FunctionHandler";
```

Comment:
The route information here doesn't matter because it's a lambda function and not an asp core function. The aws.local.operation without this change is actually just AwsLambdaFunctionName. With this change, it's aligned with other languages (AwsLambdaFunctionName + /FunctionHandler).